app/views/cases/sar_internal_review/_edit_case_details.html.slim

```diff
@@ -0,0 +1,23 @@
+- content_for :page_title, flush: true
+  = t('page_title.create_case_page.sar_internal_review.case_details_step')
```

Comment:
should  you change the title to some wording for `editing` rather than `Add`?

Reply:
Thanks good spot!